application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/views/stacktrace/StacktraceView.java

```diff
@@ -438,11 +445,13 @@ private void setReducedTree(boolean reducedTree) {
 		if (viewer instanceof TreeViewer) {
 			viewer.setContentProvider(createTreeContentProvider());
 		}
+		rebuildViewer();
```

Comment:
This should not be necessary, right?

Reply:
correct, I've forgotten to remove as I've been playing with it.